gen_mod_headers

```diff
@@ -183,6 +183,11 @@ for f in $(find . -name "Kconfig*"); do
 	copy_mkdir "$target_dir" "$f"
 done
 
+# Fix arm64 file reference
+if [ $karch = "arm64" ] && [ -e $karch_dir/include/asm/opcodes.h ]; then
```

Comment:
double-quote the expressions (`"$karch"`, and "$karch_dir/include/asm/opcodes.h"), the second one is really needed, as it would break otherwise if `karch_dir` has a space in it.

Reply:
Would probably test with `-f` as well.